compiler/semexprs.nim

```diff
@@ -2264,13 +2264,19 @@ proc semMagic(c: PContext, n: PNode, s: PSym, flags: TExprFlags): PNode =
   of mSizeOf:
     markUsed(c, n.info, s)
     result = semSizeof(c, setMs(n, s))
+  of mSetLengthSeq:
```

Comment:
This should be in ``semmagic`` because it should kick in after overload resolution.